<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPesqVeiculos</class>
 <widget class="QMainWindow" name="frmPesqVeiculos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1140</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lista de Veiculos</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Imagens/Lista_Veiculos_preto.png</normaloff>../../Imagens/Lista_Veiculos_preto.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.508, y1:0.727, x2:0.505, y2:1, stop:0 rgba(255, 255, 153, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">font: 87 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="title">
       <string>Selecione o Tipo de Pesquisa</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="cbPesquisa">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 87 10pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>Código</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Marca</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Modelo</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Disponível</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Alugado</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtPesquisa_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnPesquisa">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0, 0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}

</string>
         </property>
         <property name="text">
          <string>Pesquisar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Imagens/Pesquisar.png</normaloff>../../Imagens/Pesquisar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableWidget" name="gridVeiculos">
         <property name="styleSheet">
          <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial&quot;;</string>
         </property>
         <column>
          <property name="text">
           <string>Código</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Modelo</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Marca</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ano</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Placa</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alugado</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Batido</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Km Atual</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Valor da Diária</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descrição</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo do Veículo</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btnAlterar_2">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0, 0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}

</string>
         </property>
         <property name="text">
          <string>Alterar</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Imagens/Alterar.png</normaloff>../../Imagens/Alterar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2" alignment="Qt::AlignRight">
        <widget class="QPushButton" name="btnExcluir">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0, 0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}

</string>
         </property>
         <property name="text">
          <string>Excluir</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Imagens/lixeira.png</normaloff>../../Imagens/lixeira.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
